app/loc/lang-fi.json

```diff
@@ -81,5 +81,8 @@
     "STR_COPY": "Kopioi julkaisemattomaksi",
     "STR_CNFM_DEL_PROP": "Haluatko poistaa ehdotuksen?",
     "STR_CNFM_SEND_PROP": "Haluatko julkaista ehdotuksen?",
-    "STR_CNFM_COPY_PROP": "Haluatko kopioida ehdotuksen julkaisemattomaksi?"
+    "STR_CNFM_COPY_PROP": "Haluatko kopioida ehdotuksen julkaisemattomaksi?",
```

Comment:
I wonder if this confirmation is needed at all...oh well.
Perhaps "Luo julkaistusta ehdotuksesta uusi julkaisematon luonnos?"